src/main/kotlin/exercise3/task3/SherlockValidatesString.kt

```diff
@@ -29,7 +29,26 @@ package exercise3.task3
  */
 
 internal fun isSherlockValid(s: String): String {
-    TODO("Implement me!!!")
+    val charCount = s.groupingBy { it }.eachCount()
```

Comment:
Does your solution work for the input `accc`? 

What was the idea behind `frequencies.count { it == minFreq } == 1 && minFreq == 1` check?

Reply:
Got it :)